app/src/main/java/com/criteo/testapp/InHouseActivity.java

```diff
@@ -62,14 +59,16 @@ protected void onCreate(Bundle savedInstanceState) {
     setContentView(R.layout.activity_in_house);
     MockedIntegrationRegistry.force(Integration.IN_HOUSE);
 
-    adLayout = findViewById(R.id.AdLayout);
+    criteoBannerView = findViewById(R.id.criteoBannerView);
+    criteoBannerView.setCriteoBannerAdListener(new TestAppBannerAdListener(
+        TAG, "In-House"));
+
     btnShowInterstitial = findViewById(R.id.buttonInhouseInterstitial);
     btnShowInterstitialIbv = findViewById(R.id.buttonInhouseInterstitialIbv);
-    context = getApplicationContext();
 
     nativeLoader = new CriteoNativeLoader(
         NATIVE,
-        new TestAppNativeAdListener(TAG, NATIVE.getAdUnitId(), adLayout),
+        new TestAppNativeAdListener(TAG, NATIVE.getAdUnitId(), criteoBannerView),
```

Comment:
Shouldn't you add a native ad layout like for standalone ?
Currently it is not possible to show banner after a native.

Reply:
I must have sent the wrong patch set. I am passing the `adContainer`. Let me push again.